kmeans.py

```diff
@@ -0,0 +1,75 @@
+import random
+import math
+
+clow = 96 #minimum chest length
+chigh = 130 #maximum chest length
+hlow=73 #minimum height
+hhigh=82 #maximum height
+
+XY=[] #List of all random tuples
+C=[]  #List of randomly initializes Centroids
+index=[] #ndex representing which centroid each tuple belongs toI
+
+#Function to randomly initialize Data Samples and Centroids
```

Comment:
Comments on a function should give you a lot of information about the function. Something like this would look really good :
```
def initialize(XY,C,datanum,k):
    '''
        Generates random X values, Y values for the coordinates and also generates 'k' random centroids
        A list of random 'datanum' ( X,Y ) tuples along with 'k' ( X,Y ) centroids are populated and are returned
        @XY : Empty list for a set of ( X,Y ) coordinates
        @C : Empty list for a set of initial random centroids
        @datanum : Number of ( X,Y ) coordinates to be generated
        @k : Number of clusters to be formed
        @return : A list of randomly generated 'datanum' number of ( X,Y ) coordinated and a list of randomly generated 'k' centroids
    '''
```